src/installCommand/index.js

```diff
@@ -108,9 +109,46 @@ const installationSpec = {
         await applyPatch(packageJson, target);
       }
     },
+
+    installDidComplete: async (installedPackages) => {
+      if (!installedPackages) {
+        return;
+      }
+      await linkConditionalDependencies(installedPackages);
```

Comment:
This is needed because of how strict pnpm is — it only links those packages which are mentioned in deps. This is why we needed to modify algo to include `conditionalDependencies`.